packages/eslint-plugin/docs/rules/no-parameter-properties.md

```diff
@@ -8,7 +8,7 @@ description: 'Disallow the use of parameter properties in class constructors.'
 
 :::danger Deprecated
 
-This rule has been deprecated in favour of the equivalent, better named [`parameter-properties`](./parameter-properties) rule.
+This rule has been deprecated in favour of the equivalent, better named [`parameter-properties`](./parameter-properties.md) rule.
```

Comment:
This is already working on prod though. Why would we need to add the `.md`?

Reply:
this is not really working correctly, this generates link `/rules/no-parameter-properties/parameter-properties` instead of `/rules/parameter-properties`

Reply:
Huh, very spooky, I do repro the 404 now. But I swear I didn't before...